Tone/core/clock/TickSource.ts

```diff
@@ -3,7 +3,7 @@ import { Seconds, Ticks, Time } from "../type/Units";
 import { optionsFromArguments } from "../util/Defaults";
 import { readOnly } from "../util/Interface";
 import { PlaybackState, StateTimeline, StateTimelineEvent } from "../util/StateTimeline";
-import { Timeline } from "../util/Timeline";
+import { Timeline, TimelineEvent } from "../util/Timeline";
```

Comment:
This line seems to be triggering a linting error which is causing Travis-CI to fail. I think `TimelineEvent` is not used, so removing it should fix that linting error.

Reply:
Fixed. I'm now using the import to indicate that `TickSourceOffsetEvent`, `TickSourceTicksAtTimeEvent` and `TickSourceSecondsAtTimeEvent` are subclasses of `TimelineEvent`.